renderdoc/driver/vulkan/vk_serialise.cpp

```diff
@@ -1404,6 +1404,14 @@ static void SerialiseNext(SerialiserType &ser, VkStructureType &sType, const voi
 
     while(next)
     {
+#if ENABLED(RDOC_GGP)
```

Comment:
This isn't how platform specific extensions should be serialised - take a look at the WIN32 or ANDROID paths to see how it should be implemented.

Reply:
I think this should be done properly now.